assets/scss/templates/_guidelines.scss

```diff
@@ -85,12 +85,12 @@
 
 .color-card {
   border: 1px solid $dp-color-ghostwhite;
-  height: 200px;
+  height: 250px;
   border-radius: $dp-border-radius;
-  box-shadow: 0 0.5em 1em -0.5em $dp-color-sylver;
+  box-shadow: 0 0.5em 1em -0.5em $dp-color-silver;
   overflow: hidden;
   margin-bottom: 30px;
-  width: 212px;
+  width: 215px;
```

Comment:
This not seems to be a structural or cleaning change, it seems that you are changing the behavior here.

It is fine? what is the reason?

Reply:
Styles were cleaned and classes were added for colors, this modification was made so that the classes created are well visible in the guidelines, it is only transient. This page has to be modified, but I prefer to leave it for another pull request.

Reply:
Please leave this in the documentation in the form of a comment